### ISSUE TITLE
Translating from Python to R the implementation of EP

### ISSUE THREAD

**Issue by realkrantz** [gelman/ep-stan#3]
Hi @avehtari et al. Would it be possible to translate from Python to R the implementation of https://arxiv.org/abs/1412.4869? Thanks.

**Comment by avehtari** [comment]
Yes, there is no technical limitation. Just to clarify, are you offering to make such a translation? We are not working on such translation as there are other interesting parallelization projects going on in Stan, but we're happy to accept a pull request.
